engine/src/main/java/org/hibernate/search/query/dsl/SimpleQueryStringContext.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+
+package org.hibernate.search.query.dsl;
+
+/**
+ * @author Guillaume Smet
+ */
+public interface SimpleQueryStringContext extends QueryCustomization<SimpleQueryStringContext> {
+	/**
+	 * @param field The field name the plain text query is executed on
```

Comment:
Same here, should be about a "simple query string" rather than a "plain text query", since we decided to use that terminology.